packages/dashboard/src/layout/LayoutUtils.ts

```diff
@@ -29,6 +29,8 @@ const log = Log.module('LayoutUtils');
 
 type LayoutConfig = { id?: string; component?: string };
 
+export type PanelId = string | string[] | null | undefined;
```

Comment:
Some suggestion here to consider the `Brand` type.

```suggestion
export type PanelId = Brand<'PanelId', string | string[] | null | undefined>;
```

Reply:
Updated